packages/core/database/src/migrations/internal-migrations/5.0.0-01-convert-identifiers-long-than-max-length.ts

```diff
@@ -99,18 +94,21 @@ const renameIndex = async (knex: Knex, db: Database, diff: IndexDiff) => {
     await knex.transaction(async (trx) => {
       if (client === 'mysql' || client === 'mariadb') {
         await knex
-          .raw(
-            `ALTER TABLE \`${full.tableName}\` RENAME INDEX \`${full.indexName}\` TO \`${short.indexName}\``
-          )
+          .raw('ALTER TABLE ?? RENAME INDEX ?? TO ??', [
+            full.tableName,
+            full.indexName,
+            short.indexName,
+          ])
           .transacting(trx);
       } else if (client === 'pg' || client === 'postgres') {
         await knex
-          .raw(`ALTER INDEX "${full.indexName}" RENAME TO "${short.indexName}"`)
+          .raw('ALTER INDEX ?? RENAME TO ??', [full.indexName, short.indexName])
           .transacting(trx);
-      } else if (client === 'sqlite' || client === 'better') {
+      } else if (['sqlite', 'sqlite3', 'better-sqlite3'].includes(client as any)) {
```

Comment:
I actually think this can only be 'sqlite' but it makes zero difference here and even if it did it's only a debug log, so I'm leaving it to be safe